ci-operator/step-registry/gather/extra/gather-extra-ref.yaml

```diff
@@ -4,7 +4,7 @@ ref:
   optional_on_success: true
   from: cli
   commands: gather-extra-commands.sh
-  timeout: 900s
+  timeout: 2500s
```

Comment:
@jtaleric Let's try to make this env and configurable, if it's not already being done. That way not all tests have to wait for longer period of time but we can configure it test level.

Reply:
hm, not sure that can be done from the extra-ref? @bear-redhat is that possible?

Reply:
>  Also, we will be better off getting this change in a separate PR so we can easily get it merged by the reviewers. 

ack - I reached out to @smg247 for the review of the other components. I thought about a different PR, but since this one is hitting the timeouts, I figure we should just do it live 😄